WebUI/war/views/PercInspectionToolHandler.js

```diff
@@ -416,7 +434,7 @@
                 }
                 widgetDiv.attr('ownerId', tempContent.ownerid);
                 if (tempContent.isTransparent) {
-                    widgetDiv.addClass('perc-widget-transperant');
+                    widgetDiv.addClass('perc-widget-transparent');
```

Comment:
@ramdev1980 Is this referenced anywhere else?  You probably need to update other css / js if they were using the misspelled class name too right?

Reply:
@natechadwick  I pushed 4 more related files, based on your suggestion. Please review.